resilience4j-spring-boot2/src/test/resources/application.yaml

```diff
@@ -27,16 +27,17 @@ resilience4j.circuitbreaker:
   circuitBreakerAspectOrder: 400
   configs:
     default:
-      ringBufferSizeInClosedState: 100
-      ringBufferSizeInHalfOpenState: 10
+      slidingWindowSize: 100
```

Comment:
can we do the same for spring boo1 test resources as well

Reply:
Yes

Reply:
done